src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -4,36 +4,122 @@
 
 package frc.robot.subsystems;
 
+import static edu.wpi.first.units.Units.Inches;
+import static edu.wpi.first.units.Units.Rotations;
 import static frc.robot.Constants.IntakeConstants.*;
 
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.google.flatbuffers.Constants;
 import com.revrobotics.spark.SparkLowLevel.MotorType;
 import com.revrobotics.spark.SparkMax;
+import edu.wpi.first.units.measure.Angle;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
-import frc.robot.Constants.MotorIDConstants;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+import frc.robot.Constants.CtreCanID;
+import frc.robot.Constants.RevCanID;
+import utilities.HowdyPID;
+import utilities.TOFSensorSimple;
 
 public class IntakeSubsystem extends SubsystemBase {
   /** Creates a new IntakeSubsystem. */
   private SparkMax intakeMotor;
 
+  private TalonFX pivotMotor;
+  private SparkMax conveyorMotor;
+  private HowdyPID pivotPID;
+  private Angle pivotSetpoint = kPivotRetractAngle;
+
+  private TOFSensorSimple sensor;
+
   public IntakeSubsystem() {
-    intakeMotor = new SparkMax(MotorIDConstants.kIntakeMotor, MotorType.kBrushed);
+    intakeMotor = new SparkMax(RevCanID.kIntakeMotor, MotorType.kBrushless);
+    pivotMotor = new TalonFX(CtreCanID.kPivotMotor);
+    conveyorMotor = new SparkMax(RevCanID.kConveyorMotor, MotorType.kBrushless);
+    pivotPID = new HowdyPID(kPivotP, kPivotI, kPivotD);
+    sensor = new TOFSensorSimple(RevCanID.kConveyorSensor, Inches.of(1));
+  }
+
+  private void setPivotMotor(double speed) {
+    pivotMotor.set(speed);
+  }
+
+  private void setConveyerMotor(double speed) {
+    conveyorMotor.set(speed);
+  }
+
+  private void setIntakeMotor(double speed) {
+    intakeMotor.set(speed);
+  }
+
+  public Angle getPivotPositionCommand() {
```

Comment:
This isn't a command
```suggestion
  public Angle getPivotPosition() {
```